### PR TITLE
games-simulation/simutrans: make midi support optional

### PR DESCRIPTION
**Pull request by tastytea**
Closes: https://bugs.gentoo.org/913715

### CODE REVIEW

File: games-simulation/simutrans/simutrans-123.0.1-r2.ebuild [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+# https://simutrans-germany.com/translator/data/tab/language_pack-Base+texts.zip
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~x86"
+IUSE="minimal +midi truetype upnp zstd"
+
+DEPEND="
+	app-arch/bzip2
+	media-libs/libpng:0
```
Comment (ceamac) [inline games-simulation/simutrans/simutrans-123.0.1-r2.ebuild:33]:
```suggestion
	media-libs/libpng:0=
```

File: games-simulation/simutrans/simutrans-123.0.1-r2.ebuild [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+	media-libs/libsdl2[sound,video]
+	midi? ( media-sound/fluidsynth[sdl] )
```
Comment (ceamac) [inline games-simulation/simutrans/simutrans-123.0.1-r2.ebuild:35]:
```suggestion
	midi? ( media-sound/fluidsynth:=[sdl] )
```
This should have been moved lower, below zlib.  Unconditional dependencies are grouped together first.

**Review by ceamac — CHANGES_REQUESTED** [review]

File: games-simulation/simutrans/simutrans-123.0.1-r2.ebuild (hunk 2 above)
Comment (tastytea) [reply games-simulation/simutrans/simutrans-123.0.1-r2.ebuild]: fixed, thanks! 💖

**Comment by ceamac** [comment]
Thank you!

**Pull request closed without merging**